.example.env

```diff
@@ -0,0 +1,2 @@
+SERVER_PORT=5545
```

Comment:
This is the server's dotenv file, so it should probably be moved to `/hangoff-server` directory

Reply:
Of course...